fp/api/management/commands/crawl.py

```diff
@@ -0,0 +1,13 @@
+from scrapy.crawler import CrawlerProcess
+from django.core.management.base import BaseCommand
```

Comment:
What is the purpose of this management command?

Reply:
These are commands that are executed through django in the shell. I had added them in the past and didn't remove them. They don't use scrapyd btw.

Reply:
Okay, please remove these management commands then, they are confusing for a person who wants to use this code.

Reply:
commit 31f613d600c9dbdecaa9a18754b1979d0c2755cf resolves this